default.nix

```diff
@@ -1,22 +1,33 @@
 let
-  config = {
-    packageOverrides = super: rec {
+  thisOverlay = self: super: {
```

Comment:
I've switched this to using overlays instead of `packageOverrides`.

Almost all nix code now uses overlays, so I'd suggest you stick with that as well.